src/@carbon-info/layout/MainLayout/components/Footer/Footer.tsx

```diff
@@ -164,6 +164,17 @@ const Footer: React.FC = () => {
             This website is maintained by Switcheo Labs. The contents and opinions of this website are those of Switcheo Labs. Switcheo Labs provides links to cryptocurrency exchanges as a service to the public. Switcheo Labs does not operate these websites and is not responsible for their content and expressly rejects any liability for damages of any kind resulting from the use, reference to, or reliance on any information contained within these websites. Switcheo Labs does not warrant that the information provided by these websites is correct, complete, or up-to-date.
           </Typography>
         </Grid>
+        <Grid item md={12}>
+          <Typography className={classes.footNoteText} variant="body2" align="left">
+            {"Carbon isn't affiliated with "}
+            <Link href={"https://www.carbondefi.xyz/"} target="_blank" rel="noopener noreferrer">
+              <Typography className={classes.altCarbon} color="textSecondary" display="inline">
+                Carbon DeFi
+              </Typography>
+            </Link>
```

Comment:
Why is the font size of the link larger than the rest of the sentence?
<img width="981" alt="Screenshot 2023-07-31 at 4 12 34 PM" src="https://github.com/Switcheo/carbon-website/assets/65583522/11403a0e-bf79-4cb2-8174-23389c8446fc">